.travis.yml

```diff
@@ -19,8 +19,10 @@ env:
   - NODE_VERSION=$(cat rest-e2e-tests/.nvmrc)
   - OWNER=streamr
   - IMAGE_NAME=engine-and-editor
-  - secure: Icz6PgvdiMvqiC1yMBknaSMqHvvthOo0hUrVCTGA8pWXaqofX5fKNVI8GL4kvRK7a+YpEZ7SCvHH1cgEfBx8jC+Wc0XjGfBZXH1C1gyo8X1keCcJQVRyhxXkrjaeOLF0Cas8m9MdgAqJlwDcAXkWh426kvpAbU1SbNkkSCX3Ksp6M43LVUEbtkdOCf0GxavfgsGL5S9fk7WUf7hU4+HqL/UbbM+XMTz9hgajI5zzyRPiFwMPLGohNfopLKZvRWL/2IDY4ldVEzRi8koxIYrhO68o5RJJsptw+Gfaf3mzmgDGP9jU/cWYxsrfWogtVGFOiB2nDA+opbDU79vwHCpP5dcZJUEDvgOMVA0aTRT6Mc6hXM07wiL9noJwrSqoudkXianDDJqCKja+myKNi4wVWpisZV0HfF3nWeiKoxBaQdlCc7UUadS4OYK30Xll/Z9Z3ebWWGRmdKRSeD+QmM4j8VMu/t624AX/Q0laAS7/6BKOjMaDUqm3Q7plNWTSQWRmVUFzaH0xRL1DPLIbz/8Z8D1IVOfnq6k2FYTtVnjKQdHH65ddtqZMwtsSJsS9fQRYOm3ly0avPvDNnNeEYEuhY5lF8Tiw1lwh+BHOhiQLAPi7dqTiV/I2ImgdFH6a8wr/KICSquaSRWzwWnod3Jstrz9WWQW+K9JKu4deIRUOEx4=
-  - secure: Ro0tLYzPfBquDume3DCVOBcwGQtnkDCsitIteCjoiA75epYyweTN4pdaVdmI+nM41mKlXhbo6MYPukOVlxC0DqERsbAbQH8LKuJi+Ylio27ueC8pC0eDS+HjBu0OYLRzNuuhYalHRjAIZFI67CD1CHwFmbws6mENfte5L0sWdieOfnAmM74/82+ZpAibCC1FlIu4qLJxmHSYmH8bC5ZmNTQ59yx2h2BmsXdpJ5b2AVnoevnY8fxSb7mYC98tfS9x979+CMth510v5MzQR4AhwhmnaBcm7wjDgyINVcCG3YSQIs9nhP+AOOi8p+WCibCHGwU9uCPiDBy9KZvNppG7Wpw2JhKVSKOQvZsaxy6tarPxXEFXwaZjLbFw4aUVTe/aPsiOq+LZCljlSNFHANwVpAFqpxtUip2nap9156XNueegWDsgHHkmvp17bHRgsbk+KXvNuD2ieCiuMBJPXSKjm/6oyYsqFc9sPo/0jficsQWkegPkbBZu13acniqRZHENN91SyIh/OfZJQ5hapd8eaPS68iBVseVjplMXbaoId2PMuLw9BF1RXkc/LXHP6mN3DzrILdlezcHzdd87zJseY9+GIgEnihqvhXa53LGEB3KHe22mX0zuw7pOpaxWE89N/mSQFVehh/nnU20lylsiWpXWHe/B9vjMlmk8FnBvjuI=
+import:
```

Comment:
Please add a comment above that links to https://github.com/streamr-dev/travis-ci for easy access and since it's a new feature of Travis CI.